general_rules/Organization.tex

```diff
@@ -29,26 +26,26 @@ \subsection{Stage system}\label{rule:stages}
 
 In case of having no considerable score deviation between a team advancing to the next stage and a team dropping out, the TC may announce additional teams advancing to the next stage.
 
+Each \iterm{stage} comprehends a set of tasks grouped in two thematic scenarios.
+The \iterm{Housekeeper} scenario features tasks related to cleaning, organizing, and maintenance.
+The \iterm{Party Host} scenario focuses on providing general assistance during a party by attending the needs of the guests.
+
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Schedule}
 \label{rule:schedule}
 
 \begin{enumerate}
-	\item \textbf{Thematic scenario blocks:} Each \iterm{thematic scenario} or \iterm{theme} is split in two \iterm{blocks}.
-	At least two blocks are scheduled per day, having each block an assigned theme and lasting no less than two hours.
-	The \iaterm{Organizing Committee}{OC} announces the schedule during the setup days (see Table \ref{tbl:schedule}).
+	\item \textbf{Thematic scenario blocks:} Two blocks are scheduled per day, lasting between two and three hours.
+	All teams have assigned at least 2 \iterm{testing slots} per block in which they can test any \iterm{task} of their choice from the block's assigned scenario.
 
-	\item \textbf{Slots:} The \iaterm{Organizing Committee}{OC} assigns at least two \iterm{test slots} of 5 minutes to each team in each block.
-   The maximum number of \iterm{tests slots} will be announced during setup days by the \iaterm{Technical Committee}{TC} based on the available time and the number of participating teams.
-	A team can solve any task during its test slot.
-	Remaining block time can be used to assign additional testing slots to interested teams.
-	Testing slots are randomly assigned to teams in each block.
+	\item \textbf{Slots:} In principle, all teams get the same amount of \iterm{testing slots} with a minimum of 2 per block.
+	If there is sufficient unoccupied remaining time for all the teams in a league, a referee can open an extra testing slot, allowing an extra run for all.
 
-	\item \textbf{Tests:} Teams must inform the OC in advance which task(s) will try to solve.
+	\item \textbf{Tests:} Teams must inform the OC in advance which tasks will be tested in each block.
 	Only one task can be attempted per test slot.
 
-	\item \textbf{Participation is default:} Teams have to indicate to the \iaterm{Organizing Committee}{OC} when they are \emph{skipping} a test slot. Without such indication, they may receive a penalty when not attending (see~\refsec{rule:not_attending}).
+	\item \textbf{Participation is default:} Teams have to indicate to the \iaterm{Organizing Committee}{OC} when they are \emph{skipping} a testing slot. Without such indication, they may receive a penalty when not attending (see~\refsec{rule:not_attending}).
```

Comment:
Maybe a bit out of scope. But why 'may receive a penalty'? Just be strict.

Reply:
Traditionally we apply a penalty for not attending. It is unacceptable not be there. You have to notify OC one hour in advance.